### PR TITLE
Aqcu 990 - SVP refactor

### PR DESCRIPTION
**Pull request by kmschoep-usgs**

### CODE REVIEW

**Comment by mhines-usgs** [comment]
@kmschoep-usgs  a reminder that you can/should delete sitevisitpeak-render if you haven't already!

File: tests/testthat/test-sitevisitpeak.R [hunk 1]
```diff
@@ -4,17 +4,70 @@ wd <- getwd()
 setwd(dir = tempdir())
 
 context("testing sitevisitpeak")
+
 test_that("sitevisitpeak examples work",{
   library(jsonlite)
   
   data <- fromJSON(system.file('extdata','sitevisitpeak','sitevisitpeak-example.json', package = 'repgen'))
-  expect_is(sitevisitpeak(data,'html', 'Author Name'), 'character')
+  expect_is(sitevisitpeak(data, 'html', 'Author Name'), 'character')
```
Comment (mhines-usgs) [inline tests/testthat/test-sitevisitpeak.R]: I think you can remove 'html' now from these

Comment (thongsav-usgs) [reply tests/testthat/test-sitevisitpeak.R]: Yes, you can. This is actually passing "html" as the author name and the author name is ignored.

**Review by thongsav-usgs — APPROVED** [review]

File: tests/testthat/test-sitevisitpeak.R (hunk 1 above)
Comment (kmschoep-usgs) [reply tests/testthat/test-sitevisitpeak.R]: OK, I'll do this, then someone with write access can merge.

**Pull request merged**